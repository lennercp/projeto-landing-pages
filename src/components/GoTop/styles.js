import styled, { css } from 'styled-components';

export const Container = styled.a`
	${({ theme }) => css`
    position: fixed;
    z-index: 6;
    background: ${theme.colors.primaryColor};
    color: ${theme.colors.white};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    opacity: 0.7;
`}
`;
