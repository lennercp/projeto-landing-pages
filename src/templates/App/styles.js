import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
background: aqua;
`;
