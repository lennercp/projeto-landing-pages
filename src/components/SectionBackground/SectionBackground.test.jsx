import { renderTheme } from '../../styles/render-theme';
import { screen } from '@testing-library/react';
import { theme } from '../../styles/theme';
import { SectionBackground } from '.';

describe('<SectionBackground />', () => {
  it('should render with background dark', () => {
    const { container } = renderTheme(
      <SectionBackground background>Children</SectionBackground>,
    );

    expect(screen.getByText(/children/i)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  it('should render with background light', () => {
    const { container } = renderTheme(
      <SectionBackground light>Children</SectionBackground>,
    );

    expect(screen.getByText(/children/i)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
