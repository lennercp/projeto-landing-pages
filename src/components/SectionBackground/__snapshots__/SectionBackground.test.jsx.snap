// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionBackground /> should render with background dark 1`] = `
.c0 {
  background: #fff;
  color: #0A1128;
  background: #0A1128;
  color: #fff;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

<div>
  <div
    class="c0"
    id=""
  >
    <div
      class="c1"
    >
      Children
    </div>
  </div>
</div>
`;

exports[`<SectionBackground /> should render with background light 1`] = `
.c0 {
  background: #fff;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

<div>
  <div
    class="c0"
    id=""
  >
    <div
      class="c1"
    >
      Children
    </div>
  </div>
</div>
`;
